src/metabase/api/permissions.clj

```diff
@@ -22,9 +23,12 @@
 
 (api/defendpoint GET "/graph"
   "Fetch a graph of all Permissions."
-  []
+  [group_id]
+  {group_id (s/maybe su/IntGreaterThanZero)}
   (api/check-superuser)
-  (perms/graph))
+  (if (some? group_id)
+      (perms/graph group_id)
+      (perms/graph)))
```

Comment:
https://guide.clojure.style/#body-indentation

You have Common Lisp-style `if` indentation here... indent both the `then` and `else` forms just 2 spaces in Clojure